src/busterbunny.js

```diff
@@ -49,6 +50,15 @@ module.exports = (function() {
             EVENT_NACKED: 'event-nacked'
         });
 
+        if(config.stats){
+            function emitStats()
+            {
+                self.emit(self.EVENTS.STATS, _stats);
```

Comment:
Also, should a copy of the `_stats` be emitted instead ? The `_stats_` object may mutate in the time that it is emitted & consumed. 

@kakashif & I ran into this with another library as well where an emitted array mutated while we were processing it